tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I mostly copied this from a vite template. I don't really know what all of this means, or if any of it is redundant with the defaults